tests/t0112_gateway_cors_test.go

```diff
@@ -0,0 +1,52 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/ipfs/gateway-conformance/tooling/test"
+	. "github.com/ipfs/gateway-conformance/tooling/test"
+)
+
+func TestCors(t *testing.T) {
+	cidHello := "bafkqabtimvwgy3yk" // hello
+
+	tests := SugarTests{
+		{
+			Name: "GET Response from Gateway should contain CORS headers",
+			Request: Request().
+				Path("ipfs/{{CID}}/", cidHello),
+			Response: Expect().
+				Headers(
+					Header("Access-Control-Allow-Origin").Equals("*"),
+					Header("Access-Control-Allow-Methods").Equals("GET"),
```

Comment:
There may be a gateway where someone experiments with POST/PUT, it should not break this test.
```suggestion
					Header("Access-Control-Allow-Methods").Has("GET"),
```